Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -93,6 +95,24 @@ defineSuite([
     var uberLabelColor = Color.fromBytes(0xee, 0xee, 0xee, 0xee);
     var uberLabelScale = 4;
 
+    var oldConsoleLog;
+    var consoleMessages;
+
+    beforeAll(function() {
+        oldConsoleLog = console.log;
+        console.log = function(msg) {
```

Comment:
Check out the testing guide's section on spies: https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide#spies 

Basically, you'll want to use `spyOn` and `and.callThrough` and then you can use something like `expect(yourSpy).toHaveBeenCalledWith('expected message');`